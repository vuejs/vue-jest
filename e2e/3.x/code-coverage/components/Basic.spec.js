import { shallowMount } from '@vue/test-utils'

import Basic from './Basic.vue'

describe('Basic', () => {
  it('collects test coverage', () => {
    expect.assertions(4)

    const wrapper = shallowMount(Basic)
    const div = wrapper.find('div')
    const h1 = wrapper.find('h1')

    expect(wrapper.element).toMatchSnapshot()
    expect(div.attributes('class')).toBe('hello')
    expect(h1.attributes('class')).toBe('blue')
    expect(h1.text()).toBe('Hello World')
  })
})
