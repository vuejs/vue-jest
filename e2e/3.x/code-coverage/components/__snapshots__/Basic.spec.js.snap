// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic renders 1`] = `
<div
  class="hello"
>
  <h1
    class="blue"
  >
    Hello World
  </h1>
</div>
`;
